### PR TITLE
SPUI-415 > Bug/operation history details fix

### PR DESCRIPTION
**Pull request by VovaMedvediev**

### CODE REVIEW

File: lib/app/shared/features/wallet/view/components/wallets_body/components/transactions_list_item/components/transaction_details/buy_sell_details.dart [hunk 1]
```diff
@@ -33,6 +34,17 @@ class BuySellDetails extends StatelessWidget {
                   '${transactionListItem.swapInfo!.sellAssetId}',
             ),
           ),
+          const SpaceH10(),
+          TransactionDetailsItem(
+            text: 'Rate',
+            value: TransactionDetailsValueText(
+              text: '${transactionListItem.swapInfo!.baseRate} '
```
Comment (eli1stark) [inline lib/app/shared/features/wallet/view/components/wallets_body/components/transactions_list_item/components/transaction_details/buy_sell_details.dart:41]: definitely function

File: lib/app/shared/features/wallet/view/components/wallets_body/components/transactions_list_item/components/transaction_details/buy_sell_details.dart [hunk 2]
```diff
@@ -42,6 +54,17 @@ class BuySellDetails extends StatelessWidget {
                   '${transactionListItem.swapInfo!.buyAssetId}',
             ),
           ),
+          const SpaceH10(),
+          TransactionDetailsItem(
+            text: 'Rate',
+            value: TransactionDetailsValueText(
+                text:
+                    '${transactionListItem.swapInfo!.baseRate}'
```
Comment (eli1stark) [inline lib/app/shared/features/wallet/view/components/wallets_body/components/transactions_list_item/components/transaction_details/buy_sell_details.dart:62]: the same

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/app/shared/features/wallet/view/components/wallets_body/components/transactions_list_item/components/transaction_details/buy_sell_details.dart (hunk 1 above)
Comment (VovaMedvediev) [reply lib/app/shared/features/wallet/view/components/wallets_body/components/transactions_list_item/components/transaction_details/buy_sell_details.dart]: done

File: lib/app/shared/features/wallet/view/components/wallets_body/components/transactions_list_item/components/transaction_details/buy_sell_details.dart (hunk 2 above)
Comment (VovaMedvediev) [reply lib/app/shared/features/wallet/view/components/wallets_body/components/transactions_list_item/components/transaction_details/buy_sell_details.dart]: done

**Review by eli1stark — APPROVED** [review]

**Pull request merged**